src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs

```diff
@@ -1,77 +1,108 @@
 using System;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Localization;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Options;
+using OrchardCore.Admin;
+using OrchardCore.DisplayManagement.Descriptors;
 using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.Environment.Shell;
 using OrchardCore.Localization.Drivers;
 using OrchardCore.Localization.Models;
 using OrchardCore.Localization.Services;
 using OrchardCore.Modules;
 using OrchardCore.Navigation;
+using OrchardCore.Routing;
 using OrchardCore.Security.Permissions;
 using OrchardCore.Settings;
 using OrchardCore.Settings.Deployment;
 
-namespace OrchardCore.Localization
+namespace OrchardCore.Localization;
+
+/// <summary>
+/// Represents a localization module entry point.
+/// </summary>
+public class Startup : StartupBase
 {
-    /// <summary>
-    /// Represents a localization module entry point.
-    /// </summary>
-    public class Startup : StartupBase
+    public override int ConfigureOrder => -100;
+
+    /// <inheritdocs />
+    public override void ConfigureServices(IServiceCollection services)
     {
-        public override int ConfigureOrder => -100;
+        services.AddScoped<IDisplayDriver<ISite>, LocalizationSettingsDisplayDriver>();
+        services.AddScoped<INavigationProvider, AdminMenu>();
+        services.AddScoped<IPermissionProvider, Permissions>();
+        services.AddScoped<ILocalizationService, LocalizationService>();
 
-        /// <inheritdocs />
-        public override void ConfigureServices(IServiceCollection services)
-        {
-            services.AddScoped<IDisplayDriver<ISite>, LocalizationSettingsDisplayDriver>();
-            services.AddScoped<INavigationProvider, AdminMenu>();
-            services.AddScoped<IPermissionProvider, Permissions>();
-            services.AddScoped<ILocalizationService, LocalizationService>();
+        services.AddPortableObjectLocalization(options => options.ResourcesPath = "Localization").
+            AddDataAnnotationsPortableObjectLocalization();
 
-            services.AddPortableObjectLocalization(options => options.ResourcesPath = "Localization").
-                AddDataAnnotationsPortableObjectLocalization();
+        services.AddPortableObjectLocalization(options => options.ResourcesPath = "Localization").
+            AddDataAnnotationsPortableObjectLocalization();
```

Comment:
These 2 lines are duplicates of lines 41 and 42.

Maybe would have been better to not change the namespace line and then not refactor all this part.

Was not obvious to see these duplicates ;)

Reply:
Let me review the file, I faced some issues during the merge

Reply:
Seems this is the only place left after the merge conflict